setup.cfg

```diff
@@ -69,6 +76,11 @@ omit =
 console_scripts =
   find_pg = pgfinder.find_pg:main
 
+[options.package_data]
+* =
+  config/*.yaml
```

Comment:
Why does the interactive notebook need this file? Aren't the parameters generated by user input?